sdk/monitor/monitor-query/src/metricsQueryClient.ts

```diff
@@ -272,13 +278,13 @@ export class MetricsQueryClient {
   }
 }
 
-function formatScope(endpoint: string | undefined): string {
-  if (endpoint) {
-    if (endpoint.endsWith("/")) {
-      endpoint += "/";
+function formatScope(audience: string | undefined): string {
+  if (audience) {
+    if (audience.endsWith("/")) {
```

Comment:
So if `audience` ends with a forward slash, we add 2 more forward slashes? So `https://foo/` would end up becomming `https://foo///.default`? That seems odd, any chance this audience is documented anywhere? I'm curious about the 3 slashes.

Reply:
Hmm, I am also not sure about this now - bcz earlier the `endpoint` had the same logic.

Reply:
I tried the one with 3 forward slashes in the browser - it worked though. @jeremymeng Do you know what will be correct for this?

Reply:
I think the nearest thing I can find talking about doubling-up of '/'s being valid is here:
https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#trailing-slash-and-default

Even with that though it seems incorrect for me to touch anything in there at all - maybe we should let the customer pass in the entire string rather than checking slashes (which might not even be valid to do!) or even adding in the /.default scope.

Reply:
(Either way, @KarishmaGhiya, you're going to have to get out of this triple-slash game I somehow enrolled you in)

Reply:
Removed the `formatScope` function